CMakeLists.txt

```diff
@@ -771,7 +771,7 @@ endif()
 
 if(RUN_CLANG_TIDY)
     message(STATUS "Looking for clang-tidy")
-    find_program(CLANG_TIDY clang-tidy)
+    find_program(CLANG_TIDY clang-tidy-20)
```

Comment:
Is this change supposed to be included?

Reply:
no, looks like I committed that by accident. Thanks; reverted in 268ad52dec1f5d3cf121ce5b1e4ba5b0d9a71472